src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.star-is-active {
+  background-image: url("./images/star-active.svg");
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+}
```

remove these styles, leave only background-image
you've added them in .stars__star

```suggestion
.star-is-active {
  background-image: url("./images/star-active.svg");
}
```